portfolio.css

```diff
@@ -0,0 +1,95 @@
+* {
+	margin: 0;
+	font-size: 20px;
+	color: #ff4716;
+	overflow: auto;
+}
+
+.navbar a {
+	padding: 50% 0 0 5%;
+	text-decoration: none;
+	margin: 50% 0 50% 0;
+	position: relative;
+	top: 30px;
+	font-size: 1.5rem;	
+}
+
+.navbar {
+	background-color: #feeeeb;
+	padding: 0 0 30px;
+}
+
+.welcome-banner {
+	background-image: url("https://images.pexels.com/photos/2132171/pexels-photo-2132171.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
+	width: 100%;
+	height: 100vh;
+	text-align: center;
+	background-size: cover;
+	background-repeat: no-repeat;
+}
+
+@media screen and (max-width: 800px) {
```

Comment:
Not quite sure what Rayna's preference is, but I tend to group all my @media queries at the bottom. This way all the main CSS is together and all the variations for different screen sizes are together.

Reply:
I think so, this has now been moved to end of CSS document